types/stripe/index.d.ts

```diff
@@ -4090,20 +4090,20 @@ declare namespace Stripe {
 
         interface IBankAccountHash extends IResourceObject {
             /**
-             * value is "customer_bank_account"
+             * value is "bank_account"
              */
-            object: "customer_bank_account";
+            object: "bank_account";
```

Comment:
`bank_account` is the proper name of the object, and it is what stripe currently returns: https://stripe.com/docs/api/customer_bank_accounts/object#customer_bank_account_object-object

The only reference I could find to `customer_bank_account` is an old IRC chat that mentions it was a documentation mistake: https://webcache.googleusercontent.com/search?q=cache:RXkeQmg0xLUJ:https://botbot.me/freenode/stripe/2016-04-14/%3Fpage%3D5+&cd=3&hl=en&ct=clnk&gl=us